tips/TIP-0048/openapi3-core.yaml

```diff
@@ -3427,19 +3457,73 @@ components:
           description: The slot index of the output.
         outputIndex:
           type: integer
-          description: The index of the output.
+          description: The index of the output within the corresponding transaction.
         transactionCommitment:
           type: string
           description: The commitment of the transaction that created the output. Hex-encoded with 0x prefix.
         outputCommitmentProof:
-          type: string
           description: The proof of the output commitment. Hex-encoded with 0x prefix.
+          anyOf:
+              - $ref: '#/components/schemas/HashableNode'
+              - $ref: '#/components/schemas/LeafHash'
+              - $ref: '#/components/schemas/ValueHash'
       required:
         - slot
         - outputIndex
         - transactionCommitment
         - outputCommitmentProof
 
+    HashableNode:
+      description: Node contains the hashes of the left and right children of a node in the tree.
+      properties:
+        type:
+          type: integer
+          description: Set to value 0 to denote a Hashable Node.
+        node:
```

Comment:
Currently not a field in the API, should it be removed or the API changed?
The `l` and `r` fields are directly there

Reply:
The shown section for this comment starts at the `type`, but I meant only the `node` field. This should be removed and the "l" and "r" fields should be at this position as in your example, right?

Reply:
Yes, you're right!